### DISCUSSION TITLE
UI Test using Cypress

### DISCUSSION THREAD

**Discussion by bill-n** [finos/git-proxy#610] [General]
@JamieSlome ...Can we create some issues to expand the UI test ?

**Comment by JamieSlome** [comment]
@bill-n - absolutely! Do you have some issues and tests in mind that you think we should implement?

**Reply by bill-n** [reply]
Verify that all components and functionalities are displayed correctly on the dashboard

**Reply by JamieSlome** [reply]
@bill-n - would you be happy to open a new issue with some specifications for some tests and we can prune through and add more together?

**Reply by bill-n** [reply]
yes sure, let me get on to it
